### PR TITLE
Feature/redocusaurus and furo

### PR DESCRIPTION
**Pull request by OlaStenberg**
Added Furo API docs and API dropdown to navbar

image

See:
https://sushi-docs-rfv1c3av5-teamsushi.vercel.app/api/furo

### CODE REVIEW

**Review by 0xGenghisGoose — CHANGES_REQUESTED** [review]
Looks good, if you can make that version change Jiro mentioned & resolve the yarn.lock conflicts real quick this one will get good to go!

File: openapi/furo/v1/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,434 @@
+---
+openapi: 3.0.1
+info:
+  title: Furo
+  description: Earn, stream and automate your DAO salaries and token vesting with Furo.
+  version: "1"
```
[the remark this answers is not in the document]
Comment (OlaStenberg) [reply openapi/furo/v1/openapi.yaml:6]: Sure, agree!

**Comment by OlaStenberg** [comment]
> The side menu seems kinda finicky too, it's not moving down the page accordingly
> 
> If it's limitation with plugin or how it's setup though no biggy. Rest of it LGTM

Yeah, not sure, will have a quick look tomorrow morning

**Comment by OlaStenberg** [comment]
Yeah, seem to be a known issue a limitation in the plugin, lets roll with it for now? opened an issue at https://github.com/rohit-gohri/redocusaurus/issues/214

**Comment by jiro-ono** [comment]
Ahh I see, yeah all good let's run with it for now figure it out later

**Pull request merged**